src/components/analytics/preview.vue

```diff
@@ -0,0 +1,143 @@
+<!--
+Copyright 2021 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <modal id="analytics-preview" :state="state" hideable backdrop large @hide="$emit('hide')">
+    <template #title>{{ $t('title') }}</template>
+    <template #body>
+      <div class="modal-introduction">
+        <p>{{ $t('introduction[0]') }}</p>
+        <p>{{ $t('introduction[1]') }}</p>
+        <p>{{ $t('introduction[2]') }}</p>
+      </div>
+      <loading :state="$store.getters.initiallyLoading(['analyticsPreview'])"/>
+      <template v-if="analyticsPreview">
+        <analytics-metrics-table title="system" :metrics="systemSummary"/>
+        <div id="projectSummary">
+          <span class="header">{{ $t('projects.title') }}</span>
+          <span class="explanation">{{ $t('projects.subtitle', { numProjects }) }}</span>
+        </div>
+        <div style="display: flex">
+          <div id="usersFormsColumn">
+            <analytics-metrics-table title="users" :metrics="userSummary"/>
+            <analytics-metrics-table title="forms" :metrics="formSummary"/>
+          </div>
+          <analytics-metrics-table title="submissions" :metrics="submissionSummary"/>
+        </div>
+      </template>
+      <div class="modal-actions">
+        <button type="button" class="btn btn-primary" @click="$emit('hide')">
+          {{ $t('action.close') }}
+        </button>
+      </div>
+    </template>
+  </modal>
+</template>
+
+<script>
+import Loading from '../loading.vue';
+import Modal from '../modal.vue';
+import AnalyticsMetricsTable from './metrics-table.vue';
+
+import { noop } from '../../util/util';
+import { requestData } from '../../store/modules/request';
+
+export default {
+  name: 'AnalyticsPreview',
+  components: { AnalyticsMetricsTable, Loading, Modal },
+  props: {
+    state: Boolean
+  },
+  computed: {
+    ...requestData(['analyticsPreview']),
+    formattedJson() {
+      return JSON.stringify(this.analyticsPreview, null, 2);
+    },
+    systemSummary() {
+      return this.analyticsPreview.system;
+    },
+    firstProject() {
+      // eslint-disable-next-line arrow-body-style
+      return this.analyticsPreview.projects.reduce((a, b) => {
+        return (a.submissions.num_submissions_received.recent > b.submissions.num_submissions_received.recent) ? a : b;
+      });
+    },
```

Comment:
Because of the metrics template that is used in the backend having an empty template project in it, the API will actually return something here that is all zeroed out.